src/header/header.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './header.css';
+import Assignment from '@material-ui/icons/Assignment';
+import Hamburger from '@material-ui/icons/Menu';
+
+const header = props =>{
+  
+  let projectNameClassName = ["project-name-content"];
```

Comment:
Why didnt you use const?

Reply:
I had a thought of changing class name so i used let, i will use const here i'm pushing values to assigned object(array) not reassign object(array).